CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+If you got here it means you want to dedicate some of your time and help us with making this library better.  
+So first, thank you ❤️
+
+When contributing to this repository, please first discuss the change you wish to make via [issue](https://github.com/wix/react-native-ui-lib/issues/new/choose),
+[Discord channel](https://discord.gg/2eW4g6Z), or any other method with the owners of this repository before making a change. 
+
+## Pull Request Process
+
+1. First make sure the environment is working and synced with master. For installation details go [here](https://github.com/wix/react-native-ui-lib/blob/master/markdowns/getting-started/setup.md#demo-app)
+2. Before submitting a PR we suggest running `npm run prepush` command that verifies our lint, TS and tests were not broken.
+3. Please don't change our PR template.
+- 3.1 In Description add everything that can help the reviewer and help the reviewing process. 
+- 3.2 In the Changelog it's important a message for the library users - this message will be included in our release notes
+4. We try to make the PR review process as quick as possible, please be patient
+
+
+### Code Standards
+
+- Readability and clean code above all - we believe a clean code is easier to maintain and refactor when needed. 
+- Deciding on an API is not trivial but we always aim to keep it generic and intuitive - feel free to consult with us.
+- Our documentation is derived from our components' prop comments, make sure to add a clear description when adding new props. 
+- When possible, consider adding tets for a new functionality you add.
+
```

Comment:
1. Maybe worth adding controlled / uncontrolled preference.
2. It's possible always to have tests 🧪, at least unit (obviously not mandatory).
3. Clean code - means to separate the ui from the logic? worth adding?

Reply:
1. done, added another section in Code Standards 
2. not sure I understand, is this a question? or a suggestion to rephrase the sentence?  (: 
3. Added a bullet for that, thank

Reply:
Great :)
2. it's now more clear you referring to unit tests (`..tests for the new functionality you added`)